fourkitchens/suhumsci
# shs-4922 paragraphs image thumbnails

# READY FOR REVIEW

## Summary
This adds the Paragraphs Browser module and implements it as part of the Add Component workflow

## Steps to Test
1a. Create a flexible page and add a component to any of the following regions
 a. Full-width
 b. Main
1b. Create a basic page and add a component
2. Check that the Paragraphs Browser modal window is there
3. Selecting any of the components in the modal window will add those components to the region.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
